2200019/문자열의 거울상.py

```diff
@@ -0,0 +1,19 @@
+# qppdb
+# pqqbd
+
+T = int(input())
+
+for i in range(1,T+1):
+    t_c = str(input())
```

Comment:
조건을 나눠 잘 풀어주셨네요! 👍 
```suggestion
    t_c = input()
```
`input` 의 반환 값이 이미 string이기 때문에, `str` 함수를 적용하지 않아도 괜찮습니다! 😄